docs/variants/msi_z690/faq.md

```diff
@@ -65,22 +64,21 @@ require more recent microcode.
 
 ## Which DDR4 memory modules are supported?
 
-We can say the "supported" one is the one we have tested. It is the
-`Kingston Fury Beast KF436C17BBK4/32` part populated in all 4 slots, and you
-can check that in the [Hardware Configuration Matrix](../hardware-matrix). At
-that point it was not extensively tested either, though. Once the memory
-initialization goes through (we can see firmware logo on the display, and then
-boot into OS), we assume it is good enough at this stage.
+We can say the "supported" one is the one we have tested. It is the `Kingston
+Fury Beast KF436C17BBK4/32` part populated in all 4 slots, and you can check
+that in the [Hardware Configuration Matrix](../hardware-matrix). At that point
+it was not extensively tested either, though. Once the memory initialization
+goes through (we can see firmware logo on the display, and then boot into OS),
+we assume it is good enough at this stage.
 
-No other memory modules or combination in slots were verified so far. This is
-on the roadmap for future releases.
+No other memory modules or combination in slots were verified so far. This is on
+the roadmap for future releases.
```

Comment:
@Stanislaw-bnk przez to, że nie opublikowaliśmy nadal tego FAQ, to wg mojej wiedzy mamy już test zrobione na 3 różnych modelach kości pamięci, a nie tylko na 1. Czekamy nawet na dostawę 4-tego modelu kości RAM. @sulewskiprzemyslaw potwierdź proszę?

Reply:
@sulewskiprzemyslaw Could you update this information?